src/main/resources/db/changelog/changes/05-add-roles.yaml

```
@@ -0,0 +1,87 @@
+databaseChangeLog:
+  - changeSet:
+      id: update-roles
+      author: Dmytro
+      changes:
+        - insert:
+            tableName: roles
+            columns:
+              - column:
+                  name: role
+                  value: ADMIN
+        - insert:
+            tableName: roles
+            columns:
+              - column:
+                  name: role
+                  value: USER
+
+  - changeSet:
+      id: insert-users
+      author: Dmytro
+      changes:
+        - insert:
+            tableName: users
+            columns:
+              - column:
+                  name: id
+                  value: 1
+              - column:
+                  name: email
+                  value: user1@gmail.com
+              - column:
+                  name: password
+                  value: a4e4570c263dbfe7e7b01c451f5121f025c7bba885bb3c373ba4a578b4a7dca4
+              - column:
+                  name: first_name
+                  value: User1
+              - column:
+                  name: last_name
+                  value: Petrenko
+              - column:
+                  name: shipping_address
+                  value: "Hnata Yury street, Kiev"
+        - insert:
+            tableName: users
+            columns:
+              - column:
+                  name: id
+                  value: 2
+              - column:
+                  name: email
+                  value: user2@gmail.com
+              - column:
+                  name: password
+                  value: 3572d6025a6a37430c17cbd8356d53a3a1d394ded158811d973018f4b2d15f3b
```

```suggestion
                  value: 3572d6025a6a37430c17cbd8356d53a3a1d394ded158811d973018f4b2d15f3b #12345
```
include password in plaintext, they are unusable otherwise